OJP/OJP_StopEvents.xsd

```diff
@@ -56,6 +56,11 @@
 					<xs:documentation>Transport operators to include/exclude.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
+			<xs:element name="IncludeRestrictedLines" minOccurs="0">
+				<xs:annotation>
```

Comment:
Comment would have to be aligned if changed at the other place. Just to make sure because I don't understand how exactly it will work - Will putting the access mode on the place where the leg starts be working in all circumstances to indicate the restriction on this line?

Reply:
@trurlurl no. We have multiple scenarios:
* the ACCESS MODE restricts the LINE (e.g. dragLift)
* The needed qualification restricts the usage of the LINE: e.g. you need to be a pupil going to a given schoole, you need to stay at a given hotel, work for a given company.
But in the request we don't differenciate: They will just all show up.

Reply:
@ue71603 My concern was just that ACCESS MODE restricts the LINE only indirectly (as it is attached to a PLACE, not the line). And I'm not sure if the return structures from StopEventRequest and TripRequest always contain the initial PLACE.